sql/wsrep_server_service.cc

```diff
@@ -302,3 +302,78 @@ int Wsrep_server_service::wait_committing_transactions(int timeout)
 void Wsrep_server_service::debug_sync(const char*)
 {
 }
+
+// IV vector is the size of a block, which for AES is 16
+const unsigned int encryption_iv_size = MY_AES_BLOCK_SIZE;
```

Comment:
This constant seems to be used only once. Why not using `MY_AES_BLOCK_SIZE` directly there? Or, on the contrary, why not use the whole length of supplied IV - apparently `encryption_ctx_init()` allows passing IV of arbitrary size...